lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -387,7 +387,7 @@ def translate_treatments
             'endDate' => approximate_date(treatment['treatmentDateRange']['to']),
             'treatedDisabilityNames' => treatment['treatedDisabilityNames'],
             'center' => {
-              'name' => treatment['treatmentCenterName']
+              'name' => treatment['treatmentCenterName'].gsub(/[^a-zA-Z0-9 .()#&'"-]+/) { '' }.strip.to_s
```

Comment:
You could also write the gsub comma replacement like: gsub(/[^a-zA-Z0-9 .()#&'"-]+/, '').strip.to_s (not sure that you need a block for a simple comma replacement). Both do the same thing though 😸

Reply:
Do you forsee any other data containing invalid comma characters? Another suggestion could be to make this into a method for reuse

Reply:
@LindseySaari Awesome, I didn't know you could do that. I guess I didn't read far enough into the gsub examples.  To your second point, there are over 100 invalid text errors possible for this form. I have a feeling many are corrected via front end validation.  If we do need regex checks for mult fields I'll pull this out to a method in a future PR.

Reply:
I'd lean towards not reusing this, because their regex is goofy and I hope they don't use it widely (we don't know if they do).

Reply:
Sounds good!